packages/components/src/Accordion2/AccordionIndicator.tsx

```diff
@@ -39,6 +39,18 @@ export const AccordionIndicator = styled.div<AccordionIndicatorProps>`
   align-items: center;
   display: flex;
   justify-content: center;
+  outline: none;
+
+  ${({ focusVisible, theme }) => {
+    return (
+      focusVisible &&
+      `
+        & {
+          box-shadow: inset 0 0 0 2px ${theme.colors.keyFocus};
+        }
+      `
+    )
+  }}
```

Comment:
For readability's sake I'd pull this apart a little bit:

Extract the CSS bit as:

```
const indicatorFocusVisible = css`
  & {
    box-shadow: inset 0 0 0 2px ${({ theme }) => theme.colors.keyFocus};
  }
`
```

```suggestion
  ${({ focusVisible }) => focusVisible && indicatorFocusVisible}
```

Reply:
👍  Makes sense